website/source/docs/agent/options.html.md

```diff
@@ -533,6 +533,13 @@ default will automatically work with some tooling.
 
 #### Configuration Key Reference
 
+-> TTL values are parsed by Go's `time` package, and have the following
+[formatting specification](https://golang.org/pkg/time/#ParseDuration):
+
+> A duration string is a possibly signed sequence of decimal numbers, each
```

Comment:
This quote doesn't get formatted into the blue call-out box above, which makes it a bit hard to understand that the two are related. I think it would be ok to put the excerpt in quotes instead of a blockquote, so that it can live inside the call-out box.